docs/features/search/README.md

```diff
@@ -0,0 +1,99 @@
+---
+id: search-overview
```

Comment:
Should also update `mkdocs.yml` and `microsite/sidebars.json` for it to show up on the docs site?